lib/cobra_commander/dependencies/bundler.rb

```diff
@@ -11,7 +11,7 @@ class Bundler
       attr_reader :path
 
       def initialize(root)
-        @root = root
+        @root = Pathname.new(root)
         @path = Pathname.new(File.join(root, "Gemfile.lock")).realpath
```

Comment:
```suggestion
        @path = @root.join("Gemfile.lock").realpath
```

Reply:
Will try that once the rest of the tests are passing.